README.md

```diff
@@ -63,7 +63,11 @@ In order to add a helm chart to this repository:
 [`partner-charts-ci` documentation](https://github.com/rancher/partner-charts-ci).
 `partner-charts-ci` can be obtained by running `scripts/pull-scripts`, which
 downloads the right version for your machine to `bin/partner-charts-ci`.
-3. Create a pull request for your changes targeting the `main-source` branch in this repository.
+3. Create a pull request for your changes targeting the `main-source` branch in
+this repository. **Please ensure that your pull request only contains changes
+related to the package that you are adding.** The `PACKAGE` environment variable
+is useful for this; for more information please see the `partner-charts-ci`
```

Comment:
Maybe add the link to the `partner-charts-ci` documentation here as well?

Reply:
Specifically, link to the [example that uses the env var](https://github.com/rancher/partner-charts-ci?tab=readme-ov-file#4-run-partner-charts-ci-update)